packages/material-ui/src/Autocomplete/Autocomplete.js

```diff
@@ -78,27 +78,27 @@ export const styles = (theme) => ({
         padding: '1px 0 4px',
       },
     },
-    '&[class*="MuiOutlinedInput-root"]': {
+    '&.MuiOutlinedInput-root': {
       padding: 9,
       '$hasPopupIcon &, $hasClearIcon &': {
         paddingRight: 26 + 4 + 9,
       },
       '$hasPopupIcon$hasClearIcon &': {
         paddingRight: 52 + 4 + 9,
       },
-      '& $input': {
+      '& .MuiOutlinedInput-input': {
         padding: '7.5px 4px',
       },
-      '& $input:first-child': {
+      '& .MuiOutlinedInput-input:first-child': {
         paddingLeft: 6,
       },
       '& $endAdornment': {
         right: 9,
       },
     },
-    '&[class*="MuiOutlinedInput-root"][class*="MuiOutlinedInput-sizeSmall"]': {
+    '&.MuiOutlinedInput-root.MuiInputBase-sizeSmall': {
       padding: 6,
-      '& $input': {
+      '& .MuiOutlinedInput-input': {
         padding: '2.5px 4px',
       },
     },
```

Comment:
@natac13 see here. I'm also not sure autocomplete will work correctly until this is merged.